java/arcs/core/analysis/RecipeGraphFixpointIterator.kt

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.analysis
+
+import arcs.core.data.HandleConnectionSpec
+import arcs.core.data.Recipe
+
+/** An abstract class that implements dataflow analysis over abstract values of type [V]. */
```

Comment:
Might be worth adding an `@param` describing what's going on with the `bottom` argument

Reply:
Done.